module1.js

```diff
@@ -87,12 +90,10 @@ function question1(name1) {
   @returns the product of num1 and num2 
 */
 
-// function question5(num1, num2) {
-//   // TODO
-
-//   return /* ??? */;
-// }
-
+function question5(num1, num2) {
+  var result = num1 * num2;
+  return Math.round((result) * 100) / 100;
+};
 
```

Comment:
Well done in handling the JavaScript rounding error. You could also use the shorter...
```
return Math.round(( num1 * num2 ) * 100) / 100;
```
and save a line of code.